justfile

```diff
@@ -107,4 +107,71 @@ gh-all-prs:
   for project in "${projects[@]}"; do
     just --justfile {{justfile()}} gh-prs $project
     echo ""
-  done
\ No newline at end of file
+  done
+
+#
+# Heroku Commands
+#
+heroku-create-staging team pipeline:
+  #!/usr/bin/env bash
+  APP_NAME={{pipeline}}-staging
+  echo "Creating app '$APP_NAME' in team '{{team}}'..."
+  heroku pipelines:create {{pipeline}} --stage=staging --app=$APP_NAME --team={{team}}
+  heroku addons:create heroku-postgresql:essential-0 --app=$APP_NAME
+  heroku config:set ENVIRONMENT="staging" --app=$APP_NAME
+  heroku config:set DEBUG="True" --app=$APP_NAME
+
+heroku-create-production team pipeline:
+  #!/usr/bin/env bash
+  APP_NAME="{{pipeline}}-production"
+  echo "Creating app '$APP_NAME' in team '{{team}}'..."
+  heroku pipelines:create {{pipeline}} --stage=production --app=$APP_NAME --team={{team}}
+  heroku addons:create heroku-postgresql:standard-0 --app=$APP_NAME
+  heroku config:set ENVIRONMENT="production" --app=$APP_NAME
+  heroku config:set DEBUG="False" --app=$APP_NAME
+
+heroku-set-env-vars env_file='' app_name='':
+  #!/usr/bin/env bash
+
+  #
+  # Collects variables from a .env file and deploys.
+  #
+  # NOTES:
+  #  - This will overwrite existing variables! Confirm the correct variables are available.
+  #  - Because DB info is provisioned by Heroku, this script will ask the user if they 
+  #    want to purposefully include DB-related env vars.
+  #
+  function set_env() {
+      echo "Setting env vars from $env_file..."
+      [ -f "$env_file" ] && grep -vE '^(\s*$|#|DB_|REDIS_)' "$env_file" | tr '\n' ' ' | xargs heroku config:set -a $app_name || echo "Env file $env_file is not present, unable to set vars."
+  }
+
+  if [ -z "{{env_file}}" ]; then
+    printf "Provide the full path to a .env file, ex: ~/my-project/.env.example\n"
+    read env_file
+    env_file=$(eval echo "$env_file")
+  else
+    env_file="{{env_file}}"
+  fi
```

Comment:
It's verbose, but I think I might like this as a standard pattern for tn-cli scripts. Here this recipe allows you to either pass in your variables from the command line — in this case `env_var` and `app_name` — OR pass nothing and the script will prompt you for what it needs.

This could help with the documentation issue @oudeismetis brought up in #16. Because the prompts could give you more context on what each variable means.